### PR TITLE
[Feature | KAN-7] Added footer component

### PR DESCRIPTION
**Pull request by Jeevan1**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+import { Link } from "react-router-dom";
+import { FaLocationArrow, FaSortDown, FaSortUp } from "react-icons/fa6";
+import { MdOutlineAttachEmail, MdOutlinePhonePaused } from "react-icons/md";
+import { BiCaretDownSquare, BiCaretUpSquare } from "react-icons/bi";
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  gap: 30px;
```
Comment (RahulRimal) [inline src/components/Footer.jsx:14]: use theme her

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    font-weight: ${({ theme }) => theme.typography.fontWeight.semiBold};
+    margin-bottom: ${({ theme }) => theme.spacing.s8};
+    &:placeholder {
+      font-weight: ${({ theme }) => theme.typography.fontWeight.semiBold};
+    }
+  }
+  &::after {
+    content: "";
+    display: block;
+    width: 2px;
```
Comment (RahulRimal) [inline src/components/Footer.jsx:35]: use theme here

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+        border-bottom: 1px solid ${({ theme }) => theme.palette.primary.main};
+      }
+    }
+    &:hover ul {
+      display: block;
+    }
+  }
+`;
+
+const ContactInfo = [
```
Comment (RahulRimal) [inline src/components/Footer.jsx:135]: move this to constants.js

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+    icon: <MdOutlinePhonePaused />,
+  },
+  {
+    title: "Email",
+    value: "ccib@ccib.gmail.com",
+    icon: <MdOutlineAttachEmail />,
+  },
+];
+
+const FooterLinks = [
```
Comment (RahulRimal) [inline src/components/Footer.jsx:153]: move this to constants.js

File: src/index.css [hunk 5]
```diff
@@ -25,3 +25,16 @@ p {
 
   transition: visibility 0.3s, opacity 0.3s ease;
 }
+
+ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  margin: 0;
```
Comment (RahulRimal) [inline src/index.css:33]: add s0 on theme's spacing and use that here

File: src/index.css [hunk 6]
```diff
@@ -25,3 +25,16 @@ p {
...
+  padding: 0;
```
Comment (RahulRimal) [inline src/index.css:34]: add s0 on theme's spacing and use that here

File: src/pages/Home.jsx [hunk 7]
```diff
@@ -16,31 +16,43 @@ import useFetchTable from "../custom_hooks/useFetchTable";
...
-const Home = ({ }) => {
-
-    return (
-        <Wrapper >
-            <UsersTable />
-        </Wrapper>
-    )
-}
+  return (
+    <div style={{ margin: "20px" }}>
```
Comment (RahulRimal) [inline src/pages/Home.jsx:31]: use theme's spacing here

**Review by RahulRimal — APPROVED** [review]

File: src/components/Footer.jsx (hunk 1 above)
Comment (Jeevan1) [reply src/components/Footer.jsx:14]: done

File: src/components/Footer.jsx (hunk 2 above)
Comment (Jeevan1) [reply src/components/Footer.jsx:35]: removed width and added padding

File: src/components/Footer.jsx (hunk 3 above)
Comment (Jeevan1) [reply src/components/Footer.jsx:135]: done

File: src/components/Footer.jsx (hunk 4 above)
Comment (Jeevan1) [reply src/components/Footer.jsx:153]: done

File: src/index.css (hunk 5 above)
Comment (Jeevan1) [reply src/index.css:33]: done

File: src/index.css (hunk 6 above)
Comment (Jeevan1) [reply src/index.css:34]: done

File: src/pages/Home.jsx (hunk 7 above)
Comment (Jeevan1) [reply src/pages/Home.jsx:31]: done

File: src/index.js [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
-import ReactDOM from 'react-dom';
-import App from './App';
-import './index.css';
-import { Provider } from 'react-redux';
-import { store } from './store';
-import { SnackbarProvider } from 'notistack';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import "./index.css";
```
Comment (RahulRimal) [inline src/index.js:4]: remove this as this file has been deleted

**Review by RahulRimal — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 5 above)
Comment (Jeevan1) [reply src/index.css]: done

File: src/index.js (hunk 8 above)
Comment (Jeevan1) [reply src/index.js]: done

**Review by RahulRimal — APPROVED** [review]

**Pull request merged**